src/index.tsx

```diff
@@ -28,7 +28,12 @@ export interface MSALParams {
   authority: string;
 }
 
-export interface MSALInterativeParams extends MSALParams {
+// See https://azuread.github.io/microsoft-authentication-library-for-objc/Classes/MSALWebviewParameters.html
+export interface MSALWebViewParams {
+  privateAuthSession?: boolean; // iOS 13+
+}
```

Comment:
```suggestion
export interface MSALWebviewParams {
  prefersEphemeralWebBrowserSession?: boolean; // iOS 13+
}
```

or 

```suggestion
export interface MSALWebviewParams {
  ios_prefersEphemeralWebBrowserSession?: boolean; // iOS 13+
}
```
(Nitpick) Let's rename the interface to `MSALWebviewParams` (lowercase `view`) to be more consistent with the MSAL spelling.

I've been trying to keep parameter names as close to the native MSAL parameter names as possible. I know `prefersEphemeralWebBrowserSession` is a long name, but I think we should use it to avoid confusion.

Also, because this is an iOS only option, I'm wondering if we should prefix it with `ios_`, as we see sometimes in React Native, for example the [Switch](https://reactnative.dev/docs/switch#ios_backgroundcolor) component. Would like your thoughts on this.

Reply:
I've gone with the `ios_` prefix and updated the PR for you. Please let me know if there's anything else to look at before this can be merged.

Reply:
Looks good, thanks!